addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -174,7 +174,7 @@
                             <field name="reference" groups="account.group_account_readonly" attrs="{'readonly': [('is_ref_editable', '=', False)], 'invisible': [('product_has_cost', '=', True)]}"/>
                             <field name="date" attrs="{'readonly': [('sheet_is_editable', '=', False)]}"/>
                             <field name="accounting_date" attrs="{'invisible': ['|', ('accounting_date', '=', False), ('state', 'not in', ['approved', 'done'])]}" />
-                            <field name="account_id" options="{'no_create': True}" domain="[('internal_type', '=', 'other'), ('company_id', '=', company_id)]" groups="account.group_account_readonly" attrs="{'readonly': ['|', ('is_editable', '=', False), ('sheet_is_editable', '=', False)]}" context="{'default_company_id': company_id}"/>
+                            <field name="account_id" options="{'no_create': True}" groups="account.group_account_readonly" attrs="{'readonly': ['|', ('is_editable', '=', False), ('sheet_is_editable', '=', False)]}" context="{'default_company_id': company_id}"/>
```

Comment:
Why?

Reply:
Adding the domain seems unnecessary to me as it is the same as in the Python file (https://github.com/odoo/odoo/blob/b104a50d6d7fbc07fde8de81a4e1935e540214e5/addons/hr_expense/models/hr_expense.py#L177). I had changed it in the latest version of the PR where I simply modified the domain and I didn't touch the XML file since. Would you prefer I put it back?